src/locales/default.json

```diff
@@ -422,6 +422,12 @@
   "goToMigratedPool": "Go to migrated pool",
   "governance": "Governance",
   "grants": "Grants",
+  "greatMigration": {
+    "warning": {
+      "title": "Move your liquidity for future BAL incentives",
+      "text": "Liquidity mining incentives have expired for this pool, since upgraded pools are now live. LPs are encouraged to unstake existing positions, withdraw their liquidity and add it to an incentivized pool."
```

Comment:
```suggestion
      "text": "Liquidity mining incentives are being phased out for this pool, since upgraded pools are now live. LPs are encouraged to unstake existing positions, withdraw their liquidity and add it to an incentivized pool."
```

Reply:
done